resources/production/chainspec.toml

```diff
@@ -48,24 +48,21 @@ era_duration = '30minutes'
 # Minimum number of blocks per era.  An era will take longer than `era_duration` if that is necessary to reach the
 # minimum height.
 minimum_era_height = 10
-# Amount of time to go back before the start of the era for picking the booking block.
-booking_duration = '0ms'
-# Amount of time to wait after the booking before we pick the key block, collecting the magic bits along the way.
-entropy_duration = '0ms'
-# Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
-voting_period_duration = '2days'
-# Integer between 1 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
-# It is the percentage of validators that would need to equivocate to make two honest nodes see two conflicting
+# A number between 0 and 1 representing the fault tolerance threshold as a fraction, used by the internal finalizer.
+# It is the fraction of validators that would need to equivocate to make two honest nodes see two conflicting
 # blocks as finalized: A higher value F makes it safer to rely on finalized blocks. It also makes it more difficult to
-# finalize blocks, however, and requires strictly more than 50% + F/2 validators to be working correctly.
-finality_threshold_percent = 10
+# finalize blocks, however, and requires strictly more than (F + 1)/2 validators to be working correctly.
+finality_threshold_fraction = [1, 10]
 # Integer between 0 and 255. The power of two that is the number of milliseconds in the minimum round length, and
 # therefore the minimum delay between a block and its child. E.g. 14 means 2^14 milliseconds, i.e. about 16 seconds.
 minimum_round_exponent = 17
 # Integer between 0 and 255. Must be greater than `minimum_round_exponent`. The power of two that is the number of
 # milliseconds in the minimum round length, and therefore the minimum delay between a block and its child. E.g. 19
 # means 2^19 milliseconds, i.e. about 8.7 minutes.
 maximum_round_exponent = 19
+# The factor by which rewards for a round are multiplied if the greatest summit has ≤50% quorum, i.e. no finality.
+# Expressed as a fraction (1/5 by default).
+reduced_reward_multiplier = [1, 5]
```

Comment:
Might as well make it 1/2 for now; looks like we'll use that in mainnet: [NDRS-446](https://casperlabs.atlassian.net/browse/NDRS-446)